package.json

```diff
@@ -28,7 +28,7 @@
     "cors": "^2.8.5",
     "dotenv": "^16.0.3",
     "express": "^4.18.2",
-    "got": "11.8.3",
+    "got": "11.8.5",
```

Comment:
We should also bump the package version to `1.0.2` for this PR.  There's probably a GitHub Action that does that automatically.